src/mad_spark_alt/layers/quantitative/quality.py

```
@@ -10,6 +10,11 @@
 import torch
 from transformers import GPT2LMHeadModel, GPT2TokenizerFast
 
+from ...core.evaluation_utils import (
+    CodeAnalyzer,
+    CodeStructureMetricsDict,
```

Ruff (0.11.9)

`...core.evaluation_utils.CodeStructureMetricsDict` imported but unused

Remove unused import: `...core.evaluation_utils.CodeStructureMetricsDict`

(F401)